compositions/CollectionMenu/CollectionMenu.css.ts

```diff
@@ -24,5 +24,52 @@ export const collectionTrigger = style([
 
 export const modalWrapper = style({
   overflowY: 'scroll',
+  msOverflowStyle: 'none',
+  scrollbarWidth: 'none',
+  '::-webkit-scrollbar': {
+    display: 'none',
+  },
   height: 400,
 })
+
+export const tabsButton = style([
+  {
+    all: 'unset',
+    fontFamily: 'inherit',
+    backgroundColor: `${color.white100}`,
+    padding: '20px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    fontSize: 20,
+    color: vars.color.text.secondary,
+    cursor: 'pointer',
+    zIndex: 3,
+    width: '100%',
+    paddingBottom: space.x2,
+    borderBottom: `2px solid transparent`,
+    selectors: {
+      '&:first-child': { borderTopLeftRadius: 40 },
+      '&:last-child': { borderTopRightRadius: 40 },
+      '&:hover': { borderBottom: `2px solid ${color.black30}` },
+      '&[data-state="active"]': {
+        color: vars.color.text.primary,
+        borderBottom: `2px solid ${color.black70}`,
+      },
+    },
+  },
+])
+
+export const tabsList = style([
+  {
```

Comment:
Likewise here, display, position, and width can all be atoms.

layers.ts is the place to define zIndex values as consts so we can standardize with intention + avoid magic Z's

Reply:
Thanks, that's a great idea. I also grouped the z layers for the modal tab at 3.